src/Android/MainActivity.cs

```diff
@@ -18,7 +19,9 @@
 using Bit.Core.Utilities;
 using Bit.Droid.Receivers;
 using Bit.Droid.Utilities;
+using Xamarin.Essentials;
 using ZXing.Net.Mobile.Android;
+using FileProvider = AndroidX.Core.Content.FileProvider;
```

Comment:
I don't see this being used, if that's the case remove it

Reply:
When added the Xamarin.Essentials the FileProvider now had 2 references. Xamarin.Essentials and AndroidX.Core.Content. Keeping it as the one that was being used before